semgrep-core/tests/js/decorators_arg_method_no_body.js

```diff
@@ -0,0 +1,14 @@
+class Example {
```

Comment:
I usually prefix those tests with partial_xxx or misc_xxx
The name of the semgrep-core tests have a meaning that is used in scripts/generate-cheatsheet, so there's some
undocumented conventions there.